exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/CreateMetricDescriptorExporter.java

```diff
@@ -153,12 +153,17 @@ public void export(Collection<Metric> metrics) {
     nextExporter.export(registeredMetrics);
   }
 
-  private static boolean isSupportedExternalMetric(String metricName) {
-    for (String domain : SUPPORTED_EXTERNAL_DOMAINS) {
-      if (metricName.startsWith(domain)) {
-        return true;
-      }
+  private static boolean isBuiltInMetric(String metricName) {
+    int domainIndex = metricName.indexOf('/');
+    if (domainIndex < 0) {
+      return false;
+    }
+    String metricDomain = metricName.substring(0, domainIndex);
+    if (!metricDomain.contains("googleapis.com")) {
```

Comment:
Why not just startsWith?

Reply:
Built-in metrics are something like "bigquery.googleapis.com", "agent.googleapis.com", thus using `startsWith` "googleapis.com" won't work since the first few characters (indicate the product) can be different.

Reply:
You are correct, but just for this we can probably use endsWith.

Reply:
SGTM, updated.